pkgs/README.md

```diff
@@ -23,7 +23,18 @@ See the [CONTRIBUTING.md](../CONTRIBUTING.md) document for more general informat
 
 ## Quick Start to Adding a Package
 
-To add a package to Nixpkgs:
+Each new package has a cost for the maintainers, Hydra and users downloading
+Nixpkgs. Before adding a new package, please consider the following questions:
+* Does the package have a release?
```

Comment:
@7c6f434c what better metric for maturity do you propose? Let's restart the convo in this thread.

Reply:
Also what kind of packages do you have in mind? The vast majority of packages have releases.

Reply:
I don't think there is a good universal metric, and I don't think there can be. 

If what we care about is whether the package ready for general use, we should start by asking «Is the package ready for general use?» Sure, nothing wrong with adding «For example, if the author(s) is/are working towards the first release, it is probably too early.»

Quicklisp for one example is half-full of packages that don't bother with versions and just publish commits fixing whatever small bugs are discovered.

Reply:
Thanks, that's useful. Adopted.

Reply:
Thanks!